server/index.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const app = express();
 const router = express.Router();
+const path = require('path');
```

Comment:
Added, but not in use. Were you thinking of using `path` in your `res.sendFile` invocation?

Reply:
Yes! I will be changing what the URL looks like in my `res.sendFile()`